lib/model/sharedpullerstate.go

```diff
@@ -176,7 +177,7 @@ func (s *sharedPullerState) tempFile() (io.WriterAt, error) {
 					l.Debugln("failed to remove temporary file:", remErr)
 				}
 
-				s.failLocked("dst truncate", err)
+				s.failLocked(err)
```

Comment:
Not wrapped

Reply:
What would the context be? I expect the error message from `fd.Truncate` to say that it tried to truncate and I struggle to come up with a succinct explanation of what exactly the corner case is that resulted in this error being emitted ("setting size of sparse file with reused blocks").

Reply:
Well if it's something like permission denied, we won't know what operation it's for.

Reply:
These functions always return an error like `operation: error`, so having `dst truncate: